job_templates/power_action_-_ssh_default.erb

```diff
@@ -16,7 +16,7 @@ template_inputs:
 echo <%= input('action') %> host && sleep 3
 <%= case input('action')
       when 'restart'
-        'reboot'
+        'shutdown -r +1'
```

Comment:
How does +1 compare to now?

Reply:
"shutdown -r now" is basically the same like "reboot" - either command can trigger shutting down network *before* the command terminates, causing the "ssh" command from remote server will not receive any return value and will be waiting for it frever.